### DISCUSSION TITLE
How to get output video?

### DISCUSSION THREAD

**Discussion by ChickendCode** [ValentinH/react-easy-crop#361] [Q&A]
I seeing the example to crop video as the link below
https://codesandbox.io/s/react-easy-crop-for-videos-lfhme

However, I don't know How to get output video?
Any ideal?

Thanks.

**Comment by ValentinH** [comment]
I don't think this is possible to do it in the browser. You need to do this server side with something like Ffmpeg.

**Reply by ChickendCode** [reply]
Thank you. I am finding a solution crop video for the browser but I still have not resolved it yet. Maybe can only do it at the server
